### ISSUE TITLE
Making -Xc illegal for modern mode not good in movie

### ISSUE THREAD

**Issue by PaulWessel** [GenericMappingTools/gmt-for-geodesy#34]
Running the spinning Earth **movie** examples now gives this error:

```
gmt movie 2_earth.sh -C360p -T91 -M55,pdf -N2_earth
coast [ERROR]: Centered (-Xc) and fixed (-Xf) shifts are not available in modern mode
```
While this makes sense for modern mode in general (infinite paper size...) it makes no sense for **movie** since here we actually have a specific paper size.  So somehow we need to allow that when a **movie** script is running.  Of top of my head here are some options:

1. Let **movie** set a environmental parameter and we check for that before disallowing **-Xc -Xf**
2. Let **movie** (as it converts the _main.sh_ to the actual script that runs in parallel) detect **-Xc|f** and append an undocumented **!** which means "accept this option even in modern mode".
3. Introduce **-XC** and **-XF** for this purpose in **movie** scripts.

I think (2) is pretty simple to implement and does not require users to do anything different (and is backward compatible). Option (3) fails that.

Thoughts?

**Comment by PaulWessel** [comment]
This implies a problem for workshop running 6.4 though: The scripts that ran last year now fails.  I think the only option for the workshop is for me to use -X -Y without c but actually spell out what that center coordinates is.  Right?

**Comment by Esteban82** [comment]
Yes, it is problem. I had to change it for Indiana Jones' animations. 
But, I think that for the workshop you should/could install gmt 6.4.

**Comment by Esteban82** [comment]
Altough the users that install the dev version can have somo issues.

**Comment by PaulWessel** [comment]
I think we must expect 6.4 and not dev versions.  I will use -X6c -Y0.75c instead of -Xc -Yc for now and mention that we introduced a bug in 6.4 (actually, did not check that yet as it is not that long ago this fix happened) and that-Xc -Yc will be restored in 6.5. Also, I will mention that 6.5 allows narration track to be included.

**Comment by PaulWessel** [comment]
> Altough the users that install the dev version can have somo issues.

Dont think so; -X6c -Y0.75c works fine for everybody.

**Comment by PaulWessel** [comment]
Solution was unlisted number 4:

Check if `GMT->current.setting.ps_def_page_size` is still 32767x32767 (modern mode) or is smaller under modern mode (movie case).

Will make PR for gmt on this.
